.github/workflows/publish-docs.yml

```diff
@@ -91,6 +91,7 @@ jobs:
       DOC_AWS_SECRET_ACCESS_KEY: ${{ secrets.DOC_AWS_SECRET_ACCESS_KEY }}
     with:
       docs_directory: manufacturing-ai-suite/industrial-edge-insights-vision/apps/pallet-defect-detection
+      branch_pattern: '^main$'  # Pallet defect detection only exists on 'main' branch
```

Comment:
Hey @sathyendranv ,
Let's merge feature/eas-1.3 with latest main branch, with that hopefully all the commits appearing here should go away or at least changes should disappear